### PR TITLE
Code of Conduct

### PR DESCRIPTION
**Pull request by bourque**
This PR adds a Code of Conduct file.  The text within is borrowed from the [Contributor Covenant](https://www.contributor-covenant.org/), but we are free to tweak as we see fit.

Some things to decide on:

1. Do we want to add more detail to the `Enforcement` section? (i.e. Decide on what the consequences are for violations)
2. Is there a specific point of contact someone should go to to report a violation?
3. Do we want the same or similar CoC in the `imap_processing` repo, or any/all of the other repos under IMAP SOC?

### CODE REVIEW

**Comment by bourque** [comment]
As an aside: It looks like this PR didn't use the PR Template that you made @sdhoyt? Or did I do something wrong here?  I expected the template to automatically come up when I opened the PR but it didn't 😕

**Comment by sdhoyt** [comment]
> As an aside: It looks like this PR didn't use the PR Template that you made @sdhoyt? Or did I do something wrong here? I expected the template to automatically come up when I opened the PR but it didn't 😕

I think I need to move the PR template one directory up for it to automatically show up. It looks like the way I did it,  you need to add a query string to the URL to pick the template. Sorry about that

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (sdhoyt) [inline CODE_OF_CONDUCT.md]: Could we  use the imap.sdc-request@lists.lasp.colorado.edu email or maybe create a new one specifically for this?

Comment (greglucas) [reply CODE_OF_CONDUCT.md]: I think the main list-serve of the sdc is good. I don't think that request one will work anymore, so it should be `imap-sdc@lists.lasp.colorado.edu` I believe

**Review by greglucas — APPROVED** [review]
:+1: I think this is great. I also think the `imap_processing` repo is more likely to have external contributions, so definitely needs to go there.

**Comment by bourque** [comment]
> > As an aside: It looks like this PR didn't use the PR Template that you made @sdhoyt? Or did I do something wrong here? I expected the template to automatically come up when I opened the PR but it didn't 😕
> 
> I think I need to move the PR template one directory up for it to automatically show up. It looks like the way I did it, you need to add a query string to the URL to pick the template. Sorry about that

No worries @sdhoyt! Thanks for catching that.

**Review by sdhoyt — APPROVED** [review]

**Review by tech3371 — APPROVED** [review]
This is really good!

File: CODE_OF_CONDUCT.md
Comment (laspsandoval) [inline CODE_OF_CONDUCT.md:1]: Thank you so Matt for adding this. I am impressed. Nice job.

**Review by laspsandoval — APPROVED** [review]

**Pull request merged**